pyproject.toml

```diff
@@ -22,6 +22,7 @@ uvicorn = { extras = ["standard"], version = "^0.22.0" }
 python-magic = "^0.4.27"
 beanie = "^1.19.2"
 typer = { extras = ["all"], version = "^0.9.0" }
+jinja2 = "^3.1.2"
```

Comment:
Why is this added in the docstring ticket?

Reply:
it's not bad but it means you're pulling changes from other places which is just a bad habit